providers/bnxt_re/main.h

```diff
@@ -614,4 +620,14 @@ static inline void bnxt_re_sub_sec_busy_wait(uint32_t nsec)
 }
 
 #define BNXT_RE_HW_RETX(a) ((a)->comp_mask & BNXT_RE_COMP_MASK_UCNTX_HW_RETX_ENABLED)
+
+static inline uint32_t llog2(uint32_t x)
```

Comment:
we already have this function, "git grep ilog32"

Reply:
> we already have this function, "git grep ilog32"

Thanks. I missed searching for log32. Will update the PR.